### PR TITLE
PEAR-2023: Fix for Icons missing from dropdown menu items  

### PR DESCRIPTION
**Pull request by paribartandhakal**
## Checklist
- [N/A] Added proper unit tests
- [N/A] Left proper TODO messages for any remaining tasks
- [N/A] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

## Screenshots/Screen Recordings (if Appropriate)
Login Dropdown

### CODE REVIEW

File: packages/portal-proto/src/features/layout/LoginButtonOrUserDropdown.tsx [hunk 1]
```diff
@@ -58,12 +63,12 @@ const LoginButtonOrUserDropdown = () => {
               data-testid="usernameButton"
               ref={userDropdownRef}
             >
-              {userInfo?.data?.username}
+              {userInfo?.data?.username || "TEST"}
```
Comment (wteouchicago) [inline packages/portal-proto/src/features/layout/LoginButtonOrUserDropdown.tsx]: gotta change this...

Comment (paribartandhakal) [reply packages/portal-proto/src/features/layout/LoginButtonOrUserDropdown.tsx]: 🤦‍♂️

**Comment by wteouchicago** [comment]
icons are back! 👍

**Review by craigrbarnes — APPROVED** [review]
Updates to mantine v7 for the icons are fine.

**Pull request merged**